### DISCUSSION TITLE
Social Icons

### DISCUSSION THREAD

**Discussion by thesandeepverma** [tailwindlabs/heroicons#563] [Ideas]
We need social icons for website templates. Without social icons how can we proceed.
Now we are using social icons from fontawesome icons.
If we use heroicons then why one more another icons framework (fontawesome).
Please do needful.

**Comment by cgarvis** [comment]
Add your votes to https://github.com/tailwindlabs/heroicons/discussions/571

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
